src/components/ContentOpenWith/__tests__/ExecuteForm-test.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import ExecuteForm from '../ExecuteForm';
+
+describe('components/ContentOpenWith/ExecuteForm', () => {
+    const executePostData = {
+        url: 'foo.com',
+        params: [{ key: 'foo', value: 'bar' }, { key: 'bar', value: 'baz' }],
+    };
+
+    const submitStub = jest.fn();
+    const ref = {
+        current: {
+            submit: submitStub,
+        },
+    };
+    const onSubmitStub = jest.fn();
+
+    const getWrapper = props => shallow(<ExecuteForm {...props} />);
+
+    describe('componentDidMount', () => {
+        it('should submit the form and call callback on mount', () => {
+            React.createRef = jest.fn().mockReturnValue(ref);
+            const wrapper = getWrapper({
+                executePostData,
+                onSubmit: onSubmitStub,
+            });
+            const instance = wrapper.instance();
+
+            instance.componentDidMount();
+            expect(submitStub).toBeCalled();
```

Comment:
You could also test the values being passed back to these methods.

Reply:
These are called without arguments, not sure what you mean?

Reply:
Ah, okay. My point was just that you usually want to test that the proper values (or lack thereof) were passed to handlers.